CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -1,15 +1,19 @@
-﻿using System.Net.Http;
+﻿namespace CanHazFunny;
 
-namespace CanHazFunny
+public class JokeService : IJokeService, IJokesOut
```

Comment:
```suggestion
public class JokeService : IJokeService
```
This should be implemented in a separate class